sdk/communication/azure-communication-chat/README.md

```diff
@@ -0,0 +1,714 @@
+
+## Chat Thread
+
+A chat conversation is represented by a chat thread. Each user in the thread is called a thread member. Thread members can chat with one another privately in a 1:1 chat or huddle up in a 1:N group chat. Users also get near real-time updates for when others are typing and when they have read the messages. To learn more, read about chat concepts [here](https://docs.microsoft.com/azure/communication-services/concepts/chat/concepts).
+
+# Chat operations
+
+Once you initialize an `AzureCommunicationChatClient` class, you can perform the following chat operations:
+
+## Thread Operations
+
+- [Create a thread](#create-a-thread)
+- [Get a thread](#get-a-thread)
+- [List threads](#list-threads)
+- [Update a thread](#update-a-thread)
+- [Delete a thread](#delete-a-thread)
+
+## Message Operations
+
+- [Send a message](#send-a-message)
+- [Get a message](#get-a-message)
+- [List messages](#list-messages)
+- [Update a message](#update-a-message)
+- [Delete a message](#delete-a-message)
+
+## Thread Member Operations
+
+- [Get thread members](#get-thread-members)
+- [Add thread members](#add-thread-members)
+- [Remove a thread member](#remove-a-thread-member)
+
+## Events Operations
+
+- [Send a typing notification](#send-a-typing-notification)
+- [Send read receipt](#send-read-receipt)
+- [Get read receipts](#get-read-receipts)
+
+# Examples
+
+### Thread Operations
+
+#### Create a thread
+
+Use the `create` method to create a thread.
+
+```java
+//  The list of ChatThreadMember to be added to the thread.
+List<ChatThreadMember> members = new ArrayList<>();
+// The communication user id you created before, required.
+final String id = "<user_id>";
+// The display name for the thread member.
+final String displayName = "initial member";
+members.add(new ChatThreadMember()
+    .setId(id)
+    .setDisplayName(displayName));
+
+// The topic for the thread.
+final String topic = "General";
+// The model to pass to the create method.
+CreateChatThreadRequest thread = new CreateChatThreadRequest()
+    .setTopic(topic)
+    .setMembers(members);
+
+client.createChatThread(thread, new Callback<MultiStatusResponse>() {
+    public void onSuccess(MultiStatusResponse result, okhttp3.Response response) {
+        // MultiStatusResponse is the result returned from creating a thread.
+        // It has a 'multipleStatus' property which represents a list of IndividualStatusResponse.
+        String threadId;
+        List<IndividualStatusResponse> statusList = result.getMultipleStatus();
+        for (IndividualStatusResponse  status : statusList) {
+            if (status.getId().endsWith("@thread.v2")
+                && status.getType().contentEquals("Thread")) {
+                threadId = status.getId();
+                break;
+            }
+        }
+        // TODO: Take further action.
```

Comment:
Is it common to see these `TODO` items in code samples to imply that the user can/should do something here? The first time I read them I thought there were some pending things to add here by us.

Reply:
It seems in this document "// TODO:" is meant to say "There is more you can do" not necessarily "there is more you have to do". Not to change this interpretation, I choose not to take action on this one.